Chunk 1,3:	Correct prefix/dirs for Arch
Chunk 2,4:	Compile/link with libbsd
diff -ura docbook2mdoc-1.1.0-old/Makefile docbook2mdoc-1.1.0-new/Makefile
--- docbook2mdoc-1.1.0-old/Makefile	2019-05-02 23:37:55.000000000 +1000
+++ docbook2mdoc-1.1.0-new/Makefile	2019-05-05 16:43:54.608788611 +1000
@@ -1,7 +1,7 @@
 VERSION = 1.1.0
 CFLAGS += -g -W -Wall -Wstrict-prototypes -Wno-unused-parameter -Wwrite-strings
 WWWPREFIX = /var/www/vhosts/mdocml.bsd.lv/htdocs/docbook2mdoc
-PREFIX = /usr/local
+PREFIX = /usr
 
 HEADS =	xmalloc.h node.h parse.h reorg.h macro.h format.h
 SRCS =	xmalloc.c node.c parse.c reorg.c macro.c docbook2mdoc.c tree.c main.c
@@ -11,7 +11,7 @@
 all: docbook2mdoc
 
 docbook2mdoc: $(OBJS)
-	$(CC) -g -o $@ $(OBJS)
+	$(CC) -lbsd -g -o $@ $(OBJS)
 
 statistics: statistics.o xmalloc.o
 	$(CC) -g -o $@ statistics.o xmalloc.o
@@ -20,9 +20,9 @@
 
 install: all
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
-	mkdir -p $(DESTDIR)$(PREFIX)/man/man1
+	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
 	install -m 0755 docbook2mdoc $(DESTDIR)$(PREFIX)/bin
-	install -m 0444 docbook2mdoc.1 $(DESTDIR)$(PREFIX)/man/man1
+	install -m 0644 docbook2mdoc.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
 installwww: www
 	mkdir -p $(WWWPREFIX)/snapshots
diff -ura docbook2mdoc-1.1.0-old/node.h docbook2mdoc-1.1.0-new/node.h
--- docbook2mdoc-1.1.0-old/node.h	2019-05-02 23:37:55.000000000 +1000
+++ docbook2mdoc-1.1.0-new/node.h	2019-05-05 16:43:10.234451546 +1000
@@ -15,7 +15,7 @@
  * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
-#include <sys/queue.h>
+#include <bsd/sys/queue.h>
 
 /*
  * The interface of the DocBook syntax tree.
